goto.py

```diff
@@ -43,20 +45,23 @@ def argument_bits(self):
 
 
 def _make_code(code, codestring):
-    args = [
-        code.co_argcount,  code.co_nlocals,     code.co_stacksize,
-        code.co_flags,     codestring,          code.co_consts,
-        code.co_names,     code.co_varnames,    code.co_filename,
-        code.co_name,      code.co_firstlineno, code.co_lnotab,
-        code.co_freevars,  code.co_cellvars
-    ]
-
     try:
-        args.insert(1, code.co_kwonlyargcount)  # PY3
-    except AttributeError:
-        pass
+        return code.replace(co_code=codestring) # new in 3.8+
+    except:
```

Comment:
Please no bare `expect`s, this should be `expect AttributeError`.

Reply:
True, I'll fix